PreLoad.cmake

```diff
@@ -0,0 +1,50 @@
+# Ninja is the only supported CMake generator on Windows
```

Comment:
this preload trick was a good find!

Reply:
I don't even remember how I stumbled across this, but its a pretty handy trick! I tried it on cmake 13, 14, 15, 16 and it works in all versions.